src/App.js

```diff
@@ -22,12 +29,18 @@ export class App extends Component {
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  value={this.state.query}
+                  onChange={(event) => {
+                    this.setState({ query: event.target.value });
+                  }}
                 />
               </div>
             </div>
           </div>
 
-          <MoviesList movies={moviesFromServer} />
+          <MoviesList
+            movies={moviesFromServer.filter(this.filterFunc)}
```

Comment:
Let's move the expression `moviesFromServer.filter(this.filterFunc)` to separate variable and just pass that variable here

Reply:
++